tests/unit/test_hyper_transformer.py

```diff
@@ -1022,6 +1005,34 @@ def test_fit(self):
         ht._validate_detect_config_called.assert_called_once()
         ht._unfit.assert_called_once()
 
+    def test_fit_crashes(self):
```

Comment:
This test won't pass until the instances PR gets merged (which is why the current build fails).